### DISCUSSION TITLE
Using keybinds with i3lock

### DISCUSSION THREAD

**Discussion by 0xf0xx0** [i3/i3#4688] [Q&A] [answered]
I'm coming from cinnamon, and with cinnamon i could lock my screen and then use my keyboard buttons to dim my display and keyboard backlight. Maybe its just i3lock, but i cant seem to do that with i3. Am i missing something?

**Comment by Airblader** [comment, marked as answer]
i3lock "grabs" the keyboard, meaning that it intercepts everything. We do not support letting any keys bypass that.

**Reply by 0xf0xx0** [reply]
Ahhh ok. Is there another screen locker that does allow the special keys to go through?
